lib/cts/cases/create.rb

```diff
@@ -366,30 +373,36 @@ def get_journey_for_flagged_state(flag)
     end
 
     def responder
-      @responder ||= if !options.key?(:responder)
-                       if responding_team.responders.empty?
-                         raise "Responding team '#{responding_team.name}' has no responders."
-                       else
-                         responding_team.responders.first
-                       end
-                     else
-                       CTS::find_user(options[:responder])
-                     end
+      @responder ||= begin
+        if !options.key?(:responder)
+          if responding_team.responders.empty?
+            raise "Responding team '#{responding_team.name}' has no responders."
+          else
+            responding_team.responders.first
```

Comment:
responding_team.responders.first!

Reply:
Just re-wrote the method to make intent clearer